.github/workflows/php-tests-wp-4-7-php-5-6.yml

```diff
@@ -0,0 +1,69 @@
+name: PHP Tests
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  php-tests-wp-4-7-php-5-6:
+    name: WP 4.7, PHP 5.6
+    runs-on: ubuntu-latest
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=5s --health-timeout=2s --health-retries=3
+    env:
+      DB_HOST: 127.0.0.1
+      DB_PORT: 3306
+      MYSQL_USER: user
+      MYSQL_PASSWORD: password
+      MYSQL_DATABASE: wordpress_test
+      MYSQL_ROOT_PASSWORD: password
+      WP_VERSION: '4.7'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Verify MySQL connection
+        run: |
+          while ! mysqladmin ping -h"$DB_HOST" -P"$DB_PORT" --silent; do
+            sleep 1
+          done
+      - name: Grant database access
+        env:
+          MYSQL_TCP_PORT: ${{ env.DB_PORT }}
+        run: |
+          mysql -u root -h ${DB_HOST} -e "CREATE USER ${MYSQL_USER} IDENTIFIED BY '${MYSQL_PASSWORD}';"
+          mysql -u root -h ${DB_HOST} -e "GRANT ALL PRIVILEGES ON *.* TO ${MYSQL_USER};"
+          mysql -u root -h ${DB_HOST} -e "FLUSH PRIVILEGES;"
+      - uses: shivammathur/setup-php@v2
+        with:
+          extensions: composer, mysqli
+          php-version: '5.6'
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: |
+          echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+      - name: Composer Install
+        run: docker run --rm -v "$PWD:/app" -v "${{ steps.composer-cache.outputs.dir }}:/tmp/cache" composer install --no-interaction --no-progress # Run inside docker so composer install doesn't error...
+      - name: Set up PHP test data
+        run: tests/bin/install-wp-tests.sh ${MYSQL_DATABASE} ${MYSQL_USER} ${MYSQL_PASSWORD} ${DB_HOST}:${DB_PORT} ${WP_VERSION}
+      - name: Run Unit Tests
+        run: composer test
+        env:
+          WP_DB_HOST: ${{ env.DB_HOST }}:${{ env.DB_PORT }}
```

Comment:
This variable isn't necessary unless you wanted to substitute it above in the place of `${DB_HOST}:${DB_PORT}` but I think it's better to keep that part as-is for now.